.github/workflows/pythonpackage.yml

```
@@ -4,7 +4,10 @@
 name: Python package
 
 on:
-  [push, pull_request]
+  push:
+  pull_request:
+  schedule:
+    - cron: 25 1 3 * *
```

```suggestion
    - cron: '25 1 3 * *'
```

I think that this needs to be in quotes because `*` is a special character in YAML (https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule)